src/app/i18n/index.ts

```diff
@@ -27,6 +27,13 @@ export const languages = {
             ['sarayalth', 'https://github.com/sarayalth'],
         ],
     },
+    'ja-JP': {
+        name: 'Japanese',
```

Comment:
The name here is shown in the about this app window using `app:translation_credits` (e.g. using the non-localised name in #66:), so this should also be localised.

<img width="396" alt="Screenshot 2023-06-02 at 23 24 48" src="https://github.com/samuelthomas2774/nxapi/assets/8474065/abb22423-9c69-4af4-989d-ebee62436bbf">

(Thanks @Sarayalth for pointing this out.)